src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/EETypeCreator.cs

```diff
@@ -550,7 +550,7 @@ private static unsafe int CreateArrayGCDesc(LowLevelList<bool> bitfield, int ran
             {
                 if (numSeries > 0)
                 {
-                    *ptr-- = (short)((bitfield.Count - last + baseOffset - 2) * IntPtr.Size);
+                    *ptr-- = (short)((i - last) * IntPtr.Size);
```

Comment:
```suggestion
                    *ptr-- = (short)((bitfield.Count - last) * IntPtr.Size);
```
I think this would be easier to understand. It took me a bit of looking around to find out that `i` is always `bitField.Count` at this point.

Alternatively, add `Debug.Assert(i == bitField.Count);` to make it more obvious what's going on.

Reply:
I had `Count` there initially, but then changed to `i` to make it the same as other iterations. 
I think either way is good.